packages/react-server-dom-webpack/src/__tests__/ReactFlightDOMEdge-test.js

```diff
@@ -15,6 +15,10 @@ global.ReadableStream =
 global.TextEncoder = require('util').TextEncoder;
 global.TextDecoder = require('util').TextDecoder;
 
+// Don't wait before processing work on the server.
+// TODO: we can replace this with FlightServer.act().
+global.setTimeout = cb => cb();
```

Comment:
I'm gonna trust you that this makes sense 😆 figure we're probably going to have to rethink our testing strategy for Fizz/Flight soon regardless